### PR TITLE
EOS-27255: IEM CLI integration with Event message

### PR DESCRIPTION
**Pull request by suryakumar1024**
# Problem Statement
- New EventMessage init api takes message server endpoints and cluster_id as params.
- Need to update Existing IEM CLI to pass the required params to EventMessage.init() api.

# Design
-  Updated Existing code so as to pass required Message server endpoints and cluster_id as arguments.
-  No Change in IEM CLI signature. 
-  Required Message server endpoints and clister_id details will be identiffied from the inputed file path (cluster.conf file).

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing 
- [x] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: N
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: N
- [x] Confirm Testing was performed with installed RPM [Y/N]:  Y

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [x] Is there a change in filename/package/module or signature [Y/N]: N 
- [x] If yes for above point, Is a notification sent to all other cortx components [Y/N]: NA
- [x] Jira is updated
- [x] Check if the description is clear and explained. 
- [x] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

**Review by sachinpunadikar — APPROVED** [review]
+1

File: py-utils/src/utils/iem_framework/iem_cli.py [hunk 1]
```diff
@@ -116,7 +130,8 @@ def receive(args) -> str:
         Receives IEM Message and returns to the caller, If file[-f] is passed,
         writes message to file and returns blank string to caller
         """
-        IemCli.subscribe(component=args.source, cluster_conf=args.config)
+        endpoints, _ = IemCli._get_cluster_data(args.config)
+        IemCli.subscribe(component=args.source, message_server_endpoints = endpoints)
```
Comment (rohit-k-dwivedi) [inline py-utils/src/utils/iem_framework/iem_cli.py:134]: remove space between = operators, `message_server_endpoints=endpoints`

**Review by rohit-k-dwivedi — CHANGES_REQUESTED** [review]

File: py-utils/src/utils/iem_framework/iem_cli.py (hunk 1 above)
Comment (suryakumar1024) [reply py-utils/src/utils/iem_framework/iem_cli.py]: updated in recent patch.

**Review by rohit-k-dwivedi — APPROVED** [review]

**Review by vastradparayya — APPROVED** [review]

**Review by selvakumaar5496 — APPROVED** [review]

**Comment by sachinpunadikar** [comment]
test this please

**Comment by suryakumar1024** [comment]
This PR no more valid. Requirements changed also raised different PR for the same.

**Comment by suryakumar1024** [comment]
Re-tested this patch after rebase..

**Comment by suryakumar1024** [comment]
> test this please

Tested

**Comment by sachinpunadikar** [comment]
retested the k8s build, its success
http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Cortx-Deployment/job/Kubernetes/job/py-utils/24/

**Pull request merged**